glotaran/analysis/result.py

```diff
@@ -101,7 +97,7 @@ def scheme(self) -> Scheme:
         return self._scheme
 
     @property
-    def model(self) -> Type[Model]:
+    def model(self) -> type[Model]:
```

Comment:
```suggestion
    def model(self) -> Model:
```
If this method doesn't return the model class itself but an instance, the type needs to be `Model` instead of `type[Model]`